mkdocs_macros/plugin.py

```diff
@@ -685,7 +671,10 @@ def on_page_markdown(self, markdown, page, config,
             for func in self.pre_macro_functions:
                 func(self)
             # render the macros
-            self._raw_markdown = self.render(markdown)
+            self._raw_markdown = self.render(
```

Comment:
So you added the page  argument, for debugging purposes?

Reply:
Yes, that is necessary to print the excerpt of the page content in the error handler function.